site/src/_layoutsPlayground/component.hbs

```diff
@@ -3,14 +3,15 @@
 <head>
     <meta charset="UTF-8">
     <title>{{default title 'd3fc'}}</title>
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/d3fc.min.css" rel="stylesheet" />
+    <link href="/d3fc.css" rel="stylesheet" />
+    <!-- https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/ -->
 </head>
 <body>
     <svg id="chart" width="100%" height="300px"></svg>
 
     <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/{{ slice package.dependencies.d3 1 }}/d3.js"></script> -->
```

Comment:
I know this is old but as this isn't the full list of dependencies, and you're referencing the bundle, is it worth removing these?